app/views/about/privacy.en.html.erb

```diff
@@ -6,13 +6,12 @@
   <div class="govuk-grid-column-two-thirds">
     <div class="govuk-!-margin-bottom-9">
       <h1 class="govuk-heading-xl">Privacy notice</h1>
-      <h2 class="govuk-heading-l">Apply for Criminal Legal Aid Service</h2>
-      <p class="govuk-body">This service is provided by the
+      <p class="govuk-body">The 'Apply for criminal legal aid' service is provided by the
```

Comment:
There is a helper to get the service name `<%= service_name -%>`
Let's use it anywhere we can.